File: DigitoVerificador.php
<?php

$rol= readline("Ingrese tu rut: ");

$text = (string)$rol;

$invertido = strrev($text);
$total = 0;

for ($i = 0; $i < strlen($invertido); $i++){
    $total += (($i % 6) + 2) * $invertido[$i];
}

$modulo = ($total % 11) - 11;

echo $rol . $modulo;

File: DigitoVerificador.py

rol = str(input("Ingrese tu rut: "))

inverse = rol[::-1]
total = 0

for i in range(len(inverse)):
    total = int(total) + ((i % 6) + 2) * int(inverse[i])

modulo = (total % 11) - 11
print(rol,modulo) 

